modules/test/playwright/tests/search-experiences-web/main/sxpBlueprint.spec.ts

```diff
@@ -647,3 +651,167 @@ test.describe('Blueprint can be registered as a collection provider', () => {
 		});
 	});
 });
+
+test.describe('View links to Liferay Documentation', () => {
+	test('Blueprint and element editor have working links @LPS-147066', async ({
+		apiHelpers,
+		context,
+		editSXPBlueprintPage,
+		page,
+		sxpBlueprintsAndElementsViewPage,
+	}) => {
+		const sxpBlueprint =
+			await apiHelpers.searchExperiences.createSXPBlueprint();
+
+		const assertNewPageTitle = async (title: string) => {
+			const pagePromise = context.waitForEvent('page');
+
+			const newTab = await pagePromise;
+
+			await newTab.waitForLoadState();
+
+			await expect(newTab).toHaveTitle(new RegExp(title));
+
+			await newTab.close();
+		};
```

Comment:
This seems generic enough that you could move this into a new folder `search-experiences-web/main/utils/assertNewPageTitle.ts`